src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -683,14 +683,15 @@ private void EnsureParentsAndChildren(List<string> simpleNames)
             private MetadataNode GetOrCreateChildNode(
                MetadataNode currentNode, string simpleName)
             {
-                foreach (var childNode in _parentToChildren[currentNode])
-                {
-                    if (childNode.Name == simpleName)
-                    {
-                        // Found an existing child node.  Just return that and all 
-                        // future parts off of it.
-                        return childNode;
-                    }
+                if (_parentToChildren.TryGetValue(
+                    currentNode,
```

Comment:
Indent :-)

Reply:
It is :)

Reply:
indent again.  :)

this makes it look as if this is a sibling expr to `_parentToChildren` as opposed to being logicall and physically parented by the expression containing that node.

Reply:
I recommend filing a bug for this if you aren't happy with the way it looks. This form is what I'm used to seeing, plus the current change aligns with all our current IDE feature behaviors.

Reply:
The code is infinitely with the ide formatting patterns we use on the rest of the code. This is a deal breaker for me :-)

Reply:
I reviewed multiple alternatives but was not able to find one that was clearly better.

### Unwrapping the argument list

Extracting `predicate` to a local variable allows the argument list to be unwrapped.

```csharp
Func<MetadataNode, string, bool> predicate = static (childNode, simpleName) => childNode.Name == simpleName;
if (_parentToChildren.TryGetValue(currentNode, predicate, simpleName, out var childNode))
```

Unfortunately, this approach introduces diagnostics IDE0039 (Use local function), and doing so would reduce performance. Adding a suppression plus documentation to avoid that route ended up worse than the original.

### Nested wrapping

Wrapping the member access change allows the argument list to be nested an additional level relative to the `if` token.

```csharp
if (_parentToChildren
    .TryGetValue(
        currentNode,
        static (childNode, simpleName) => childNode.Name == simpleName,
        simpleName,
        out var childNode))
```

To me, this did not improve readability or make notable progress against the original request.

Reply:
I have to agree with Cyrus. The formatting here is not readable to me at all.

Reply:
And fwiw, the format I would naturally use here is:
```cs
if (_parentToChildren.TryGetValue(
        currentNode,
        static (childNode, simpleName) => childNode.Name == simpleName,
        simpleName,
        out var childNode))
{
```

Reply:
I amended the commit to unwrap the arguments to one line.